config/certmanager/certmanager_role.yaml

```diff
@@ -11,54 +11,20 @@ rules:
     resources:
       - secrets
     verbs:
-      - list
       - watch
+      - list
       - get
       - create
-      - delete
       - update
+      - delete
   - apiGroups:
       - cert-manager.io
     resources:
       - certificates
     verbs:
-      - patch
-      - list
       - watch
-      - get
-      - create
-      - delete
-  - apiGroups:
-      - cert-manager.io
-    resources:
-      - issuers
-    verbs:
       - list
-      - watch
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: controller-manager-certmanager
-  namespace: kcp-system
-rules:
-  - apiGroups:
-      - cert-manager.io
-    resources:
-      - certificates
-    verbs:
-      - create
-      - delete
       - get
-      - list
+      - create
```

Comment:
FYI: create needed as it seems the `patch` of a non-existing object (upsert) is not working without `create`